config/manifests/bases/machine-deletion-remediation.clusterserviceversion.yaml

```diff
@@ -40,10 +40,24 @@ spec:
       - displayName: Template
         path: template
       version: v1alpha1
-  description: This operator follows the annotation on the Node to the associated
-    Machine object, confirms that it has an owning controller (e.g. MachineSetController),
-    that will recreate the Machine once deleted, and then proceeds with the deletion
-    of the Machine.
+  description: |
+    Machine Deletion Remediation (MDR) is a remediator designed to reprovision unhealthy
+    nodes using the Machine API. MDR can be used together with NodeHealthCheck (NHC),
+    which detects the unhealthy Node and creates the Custom Resource MDR needs to remediate it.
+
+    # How does it work
+    MDR follows the annotation on the Node to the associated Machine object, confirms
+    that it has an owning controller (e.g. MachineSetController), that will recreate the
+    Machine once deleted, and then proceeds with deleting the Machine.
+
+    # How to use MDR with NHC
+    The Custom Resource that MDR needs to remediate the Machine can be automatically
```

Comment:
`The Custom Resource that MDR needs to remediate the Machine` 
IMO this is a bit unclear.
- I think that we need to be consistent as on what MDR remediates Machine or Node (IMO Node).
- I would also rephrase a bit for more clarity, maybe something like `The Custom Resource that MDR needs in order to remediate the ...`